README.md

```diff
@@ -1,8 +1,8 @@
-# Mesa-Geo: a GIS extension for the Mesa agent-based modeling framework in Python
+# Mesa-Geo: GIS Extension for Mesa
```

Comment:
People might confuse this with this [Mesa](https://github.com/mesa3d/mesa).

Reply:
How about **Mesa-Geo: GIS Extension for Mesa Agent-Based Modeling**?

I wanted a shorter title because the long one doesn't look very nice on the top navigation bar at Read the Docs.

Reply:
Yes that is more informative.

Reply:
Thanks. It's now updated.